bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,330 @@
+previous step. 
+
+Run this command:
+
+```shell
+babylond create-bls-key <address>
+```
+
+Replace `<address>` with your address from the earlier keyring 
+generation (it should look similar to `bbn1kvajzzn6gtfn2x6ujfvd6q54etzxnqg7pkylk9`). 
+
+The above command will:
+1. Generate a new BLS key pair
+2. Associate it with your validator address
+3. Store it in your node's configuration file at 
+`~/.<path>/config/priv_validator_key.json`
+
+This key will be used automatically by your validator software when it needs 
+to participate in epoch-end signature collection. The BLS signatures help 
+create compact, efficient proofs of consensus that can be later timestamped to Bitcoin.
+
+> ⚠️ **Important**: The `priv_validator_key.json` file contains sensitive key 
+> material. Make sure to backup this file and store it securely, as it's 
+> essential for your validator's operation and cannot be recovered if lost.
+
+## 4. CometBFT Validator Configuration
+
+Next, we need to request the bls public key in order to create the validator 
+configuration file.
+
+```shell
+babylond tendermint show-validator --home <home>
+```
+
+It will output a public key in the following format:
+
+```shell
+{"@type":"/cosmos.crypto.ed25519.PubKey","key":"0Wlt7ZPl0uvv7onsw4gP8FSQJUk986zMcOdWselDPM4="}
+```
+
+Use the output of the command above and replace the `pubkey` value in the example below. 
+Then subsequently run the following command to create the validator configuration file:
+
+```shell
+cat > <home>/config/validator.json << EOF
+{
+  "pubkey": {"@type":"/cosmos.crypto.ed25519.PubKey","key":"0Wlt7ZPl0uvv7onsw4gP8FSQJUk986zMcOdWselDPM4="},
+  "amount": "1000000ubbn",
+  "moniker": "my-validator",
+  "commission-rate": "0.10",
+  "commission-max-rate": "0.20",
+  "commission-max-change-rate": "0.01",
+  "min-self-delegation": "1"
+}
+EOF
+```
+
+Parameters:
+- `pubkey`: Your validator's public key
+- `amount`: Initial self-delegation amount
+- `moniker`: Your validator's name/identifier
+- `commission-rate`: Current commission rate 
+- `commission-max-rate`: Maximum commission rate allowed
+- `commission-max-change-rate`: Maximum daily commission change rate
+- `min-self-delegation`: Minimum amount you must keep self-delegated
+
+If you prefer to add this manually or are having issues, another option is to 
+create a `validator.json` file and then paste the above json into it but remember
+to replace the `pubkey` value with your actual validator public key.
+
+## 5. Creating a Validator
+
+> ⚠️ **Important**: You will need a funded account for this step
+
+Unlike traditional Cosmos SDK chains that use the `staking` module, 
+Babylon uses the [`checkpointing`](https://docs.babylonlabs.io/docs/developer-guides/modules/checkpointing) module for validator creation and management.
+
+The creation process requires your previously generated BLS key, 
+which should be located at `<path>/config/priv_validator_key.json`, 
```

Comment:
This file contains both the validator and the BLS key. Why does the process require the BLS key specifically?